### DISCUSSION TITLE
Rerun location estimates?

### DISCUSSION THREAD

**Discussion by contagon** [photoprism/photoprism#1986] [Q&A] [answered]
I've been importing all our photos into photoprism, and have done a bunch of my wives/mine photos from before we met, which are naturally in different locations and often on the same day.

This has caused issues with the automatic location estimation as many of my photos are tagged with her locations and vice versa. This feature is awesome though, and I'd like it to run on at least one set of our photos. The current idea is to remove my photos, rerun estimates on my wife's photos, then import mine back in (likely with estimates turned off).

A few questions before I embark. Are archived photos considered when doing location estimation? And is there a way to run location estimation on photos that are already in photoprism? I tried indexing, but it doesn't seem to do it.

Thanks so much!

**Comment by lastzero** [comment, marked as answer]
Try

```
docker-compose exec photoprism photoprism optimize -f
```
